File: Python/0412-fizz-buzz.py
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        res = []
        dict = {
            3: "Fizz",
            5: "Buzz"
        }

        for i in range(1, n + 1):
            tmp = ""
            for key in dict.keys():
                if i % key == 0:
                    tmp += dict[key]
            if not tmp:
                tmp = str(i)
            res.append(tmp)
        return res
        

File: Java/0412-fizz-buzz.java
class Solution {
    public List<String> fizzBuzz(int n) {
        List<String> res = new ArrayList<String>();
        
        HashMap<Integer, String> hashmap = new HashMap<Integer, String>();
        hashmap.put(3, "Fizz");
        hashmap.put(5, "Buzz");

        for(int i = 1; i <=n; i++){
            String tmp = "";
            for(Integer key : hashmap.keySet()){
                if(i % key == 0)
                    tmp += hashmap.get(key);
            }
            if(tmp.equals(""))
                tmp += Integer.toString(i);
            res.add(tmp);
        }
        return res;
    }
}